dashboard/app/entities.go

```diff
@@ -71,16 +71,20 @@ type Build struct {
 }
 
 type Bug struct {
-	Namespace      string
-	Seq            int64 // sequences of the bug with the same title
-	Title          string
-	MergedTitles   []string // crash titles that we already merged into this bug
-	AltTitles      []string // alternative crash titles that we may merge into this bug
-	Status         int
-	DupOf          string
-	NumCrashes     int64
-	NumRepro       int64
+	Namespace    string
+	Seq          int64 // sequences of the bug with the same title
+	Title        string
+	MergedTitles []string // crash titles that we already merged into this bug
+	AltTitles    []string // alternative crash titles that we may merge into this bug
+	Status       int
+	StatusReason dashapi.BugStatusReason // e.g. if the bug status is "invalid", here's the reason why
+	DupOf        string
+	NumCrashes   int64
+	NumRepro     int64
+	// ReproLevel is the best ever found repro level for this bug.
+	// HeadReproLevel is best known repro level that still works on the HEAD commit.
 	ReproLevel     dashapi.ReproLevel
+	HeadReproLevel dashapi.ReproLevel `datastore:"head_repro_level"`
```

Comment:
The custom name `datastore:"head_repro_level"` looks inconsistent with the rest.
We have 99% of fields named one way and few new fields named differently. I don't think it's a good idea to change conventions at this point.
Having the same name in the code and queries also useful for grepping.